consensus/src/round_manager.rs

```diff
@@ -403,17 +403,19 @@ impl RoundManager {
             let safety_rules = self.safety_rules.clone();
             let proposer_election = self.proposer_election.clone();
             tokio::spawn(async move {
-                if let Err(e) = Self::generate_and_send_proposal(
-                    epoch_state,
-                    new_round_event,
-                    network,
-                    sync_info,
-                    proposal_generator,
-                    safety_rules,
-                    proposer_election,
-                )
-                .await
-                {
+                if let Err(e) = monitor!(
+                    "generate_and_send_proposal",
```

Comment:
Does this add more info than the existing "Block creation to signed", or is it redundant?

```
rate(aptos_consensus_block_tracing_sum{chain_name=~"$chain_name", cluster=~"$cluster", metrics_source=~"$metrics_source", namespace=~"$namespace", kubernetes_pod_name=~"$kubernetes_pod_name", role=~"$role", stage="signed"}[$interval]) / rate(aptos_consensus_block_tracing_count{chain_name=~"$chain_name", cluster=~"$cluster", metrics_source=~"$metrics_source", namespace=~"$namespace", kubernetes_pod_name=~"$kubernetes_pod_name", role=~"$role", stage="signed"}[$interval])
```